packages/backend-core/src/tenancy/db.ts

```diff
@@ -1,6 +1,24 @@
 import { getDB } from "../db/db"
 import { getGlobalDBName } from "../context"
+import { TenantInfo } from "@budibase/types"
 
 export function getTenantDB(tenantId: string) {
   return getDB(getGlobalDBName(tenantId))
 }
+
+export async function saveTenantInfo(tenantInfo: TenantInfo) {
+  const db = getTenantDB(tenantInfo.id)
+  try {
+    // save the tenant info to db
+    await db.put({
+      _id: "tenant_info",
+      ...tenantInfo,
+    })
+  } catch (err: any) {
+    if (err.status === 409) {
+      throw "Tenant info exists already"
```

Comment:
The wording of this is leaving me with questions. Is tenant info write-once? Should it never update?

Reply:
No longer catching the error.